data/SpeciesToOWGfx.c

```diff
@@ -32,474 +32,474 @@ const u16 UNUSED sSpeciesToOWGfx[] =
-    [SPECIES_CHINGLING          ] =  537,
-    [SPECIES_STUNKY             ] =  538,
-    [SPECIES_SKUNTANK           ] =  539,
-    [SPECIES_BRONZOR            ] =  540,
-    [SPECIES_BRONZONG           ] =  541,
-    [SPECIES_BONSLY             ] =  542,
-    [SPECIES_MIME_JR            ] =  543,
-    [SPECIES_HAPPINY            ] =  544,
-    [SPECIES_CHATOT             ] =  545,
-    [SPECIES_SPIRITOMB          ] =  546,
-    [SPECIES_GIBLE              ] =  547,
-    [SPECIES_GABITE             ] =  548,
-    [SPECIES_GARCHOMP           ] =  549,
-    [SPECIES_MUNCHLAX           ] =  550,
-    [SPECIES_RIOLU              ] =  551,
-    [SPECIES_LUCARIO            ] =  552,
-    [SPECIES_HIPPOPOTAS         ] =  553,
-    [SPECIES_HIPPOWDON          ] =  555,
-    [SPECIES_SKORUPI            ] =  557,
-    [SPECIES_DRAPION            ] =  558,
-    [SPECIES_CROAGUNK           ] =  559,
-    [SPECIES_TOXICROAK          ] =  560,
-    [SPECIES_CARNIVINE          ] =  561,
-    [SPECIES_FINNEON            ] =  562,
-    [SPECIES_LUMINEON           ] =  563,
-    [SPECIES_MANTYKE            ] =  564,
-    [SPECIES_SNOVER             ] =  565,
-    [SPECIES_ABOMASNOW          ] =  566,
-    [SPECIES_WEAVILE            ] =  567,
-    [SPECIES_MAGNEZONE          ] =  568,
-    [SPECIES_LICKILICKY         ] =  569,
-    [SPECIES_RHYPERIOR          ] =  570,
-    [SPECIES_TANGROWTH          ] =  571,
-    [SPECIES_ELECTIVIRE         ] =  572,
-    [SPECIES_MAGMORTAR          ] =  573,
-    [SPECIES_TOGEKISS           ] =  574,
-    [SPECIES_YANMEGA            ] =  575,
-    [SPECIES_LEAFEON            ] =  576,
-    [SPECIES_GLACEON            ] =  577,
-    [SPECIES_GLISCOR            ] =  578,
-    [SPECIES_MAMOSWINE          ] =  579,
-    [SPECIES_PORYGON_Z          ] =  580,
-    [SPECIES_GALLADE            ] =  581,
-    [SPECIES_PROBOPASS          ] =  582,
-    [SPECIES_DUSKNOIR           ] =  583,
-    [SPECIES_FROSLASS           ] =  584,
-    [SPECIES_ROTOM              ] =  585,
-    [SPECIES_UXIE               ] =  591,
-    [SPECIES_MESPRIT            ] =  592,
-    [SPECIES_AZELF              ] =  593,
-    [SPECIES_DIALGA             ] =  594,
-    [SPECIES_PALKIA             ] =  596,
-    [SPECIES_HEATRAN            ] =  598,
-    [SPECIES_REGIGIGAS          ] =  599,
-    [SPECIES_GIRATINA           ] =  600,
-    [SPECIES_CRESSELIA          ] =  602,
-    [SPECIES_PHIONE             ] =  603,
-    [SPECIES_MANAPHY            ] =  604,
-    [SPECIES_DARKRAI            ] =  605,
-    [SPECIES_SHAYMIN            ] =  606,
-    [SPECIES_ARCEUS             ] =  608,
+    [SPECIES_RAICHU             ] =   44,
+    [SPECIES_SANDSHREW          ] =   46,
+    [SPECIES_SANDSLASH          ] =   48,
+    [SPECIES_NIDORAN_F          ] =   50,
+    [SPECIES_NIDORINA           ] =   51,
+    [SPECIES_NIDOQUEEN          ] =   52,
+    [SPECIES_NIDORAN_M          ] =   53,
+    [SPECIES_NIDORINO           ] =   55,
```

Comment:
why does nidoran male have a form?

Reply:
byproduct of me incrementing everything with a big select and vsc code plug-in lol . will fix it